consensus/parlia/fusionfork.go

```diff
@@ -0,0 +1,43 @@
+package parlia
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/core/systemcontracts"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+func (p *Parlia) updateValidatorSet(blockHash common.Hash) error {
```

Comment:
This is a contract call but not a transaction, what we need is a transaction but not simply a call. 

Need refactor this part.

Reply:
Done